src/index.scss

```diff
@@ -0,0 +1,235 @@
+@mixin center {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+}
+
+body {
+  margin: 0;
+  color: #ffeead;
+  background: #ffeead;
+}
+
+.containter {
+  display: grid;
+  grid-template-areas:
+    'logo     header'
+    'sidebar  content'
+    'sidebar  content'
+    'footer   footer';
+  grid-template-rows: 50px minmax(calc(50vh - 75px), 100%) minmax(calc(50vh - 75px), 100%) 50px;
+  grid-template-columns: 150px 1fr;
+  grid-gap: 5px;
+  min-height: calc(100vh - 30px);
+  margin: 15px;
+  font-family: monospace;
+  font-size: 22px;
+  font-weight: 600;
+
+  &__logo {
+    grid-area: logo;
+    background: #96ceb4;
+
+    &-title {
+      @include center;
+
+      text-transform: uppercase;
+    }
+  }
+
+  &__header {
+    grid-area: header;
+    background: #ff706a;
+  }
+
+  &__content {
+    grid-area: content;
+    background: #fecc5d;
+  }
+
+  &__sidebar {
+    grid-area: sidebar;
+    background-color: #00cc9a;
+  }
+
+  &__footer {
+    grid-area: footer;
+    background-color: #df9edf;
+
+    &-title {
+      @include center;
+
+      margin: 0;
+      text-transform: uppercase;
+    }
+  }
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 10px;
+  text-transform: uppercase;
+
+  &__list {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__item {
+    padding: 10px 0;
+    list-style: none;
+  }
+
+  &__link {
+    padding: 0 10px;
+    color: #ffeead;
+    text-decoration: none;
+
+    &:hover {
+      text-decoration: underline;
+      cursor: pointer;
+    }
+  }
+}
+
+.sidebar {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  padding: 10px 0;
+  text-align: center;
+  text-transform: uppercase;
+
+  &__menu {
+    margin: 10px 0 0;
+    color: #ffeead;
+    text-decoration: none;
+  }
+
+  &__list {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__item {
+    padding: 10px 0;
+    list-style: none;
+  }
+
+  &__link {
+    color: #ffeead;
+    text-decoration: none;
+    text-transform: capitalize;
+
+    &:hover {
+      text-decoration: underline;
+      cursor: pointer;
+    }
+  }
+}
+
+.content {
+  display: grid;
+  grid-template-areas:
+    'one    two    four'
+    'three  three  four';
+  grid-template-rows: 1fr 1fr;
+  grid-template-columns: 1fr 3fr 2fr;
+  grid-gap: 10px;
+  padding: 30px;
+  text-align: center;
+  text-transform: uppercase;
+
+  &__one {
+    box-sizing: border-box;
```

Comment:
better  box-sizing: border-box; to body. Will be useful to have this property on all elements.

Reply:
ok

Reply:
in body not working, but use `*` I dont want

Reply:
why not using * ?

Reply:
I have setting in stylelint. I thought that `*` it is not good practice

Reply:
From one point of view yes. But from another you have to add border-box to certain elements which is not useful at all. In this layout I completely accept your solution. But in a real project I would not agree with you.

Reply:
In real project I setup normalize.scss or reset.scss